docs/loading-in-profiles.md

```diff
@@ -21,16 +21,77 @@ uploadprofile /path/to/profile.js
 
 > `https://perf-html.io/from-url/{URL}`
 
-Profiles can also be loaded in from arbitrary URLs. In addition, you can then upload it to the online storage directly from perf.html's interface.
+Profiles can also be loaded in from arbitrary URLs. In addition, you can then upload it to the online storage directly from perf.html's interface. The profile needs to be served from https, and have the appropriate `Access-Control-Allow-Origin` header set to either `*` or `https://perf-html.io`.
 
 See the function below for an easy utility for converting to the proper URL format. Here is a simple utility function to turn a profile URL into a perf.html URL to view it:
 
 ```js
 function getPerfHtmlUrl (profileUrl) {
-  return `https://perf-html.io/from-url/${encodeURIComponent(profileUrl)}`;
+  return "https://perf-html.io/from-url/" + encodeURIComponent(profileUrl)};
 }
 ```
 
+Or in Python:
+
+```python
+import urllib
+
+def get_perf_html_url(url):
+    return "https://perf-html.io/from-url/" + urllib.quote(url, safe="")
+```
+
+Here is an example profile server written in Node.js:
+
+```js
+const https = require('https');
+const fs = require('fs');
+const path = require('path');
+const querystring = require('querystring');
+
+const PORT = 3000;
+
+// If loading in a self-signed certificate, you must first manually add an exception
+// to it in the browser, or else perf.html will throw a NetworkError error.
```

Comment:
This seems like a major hassle. I thought localhost was treated as a secure origin. Can we do something about this?

Reply:
I agree it's a major hassle, and it might be a blocker for Bug 1425308. I'm not seeing anything with a quick googling.